### ISSUE TITLE
print_with_newline: false positive

### ISSUE THREAD

**Issue by letheed** [Manishearth/rust-clippy#1275]
The following code triggers a false positive `print_with_newline` lint:
```
print!("\n{}", foo);
```

**Comment by mcarton** [comment]
Thanks for the report! This has been fixed in #1268 but has not been published yet.

**Comment by letheed** [comment]
Oh, great! I didn't check  the closed bugs.
Thanks
